CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.6.3)
```

Comment:
is this the cmake version used by KVS team?

Reply:
Yes, this version is same as [WebRTC-C-SDK one](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/master/CMakeLists.txt#L1).